package.json

```diff
@@ -24,11 +24,12 @@
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jsx-a11y": "^5.1.1",
     "eslint-plugin-react": "^7.1.0",
-    "gulp": "^3.9.1",
-    "mocha": "^3.5.0",
+    "gulp": "^4.0.0",
```

Comment:
Modifying the version of Gulp or other critical packages should be separated to their own pull request as this will require extensive testing to ensure our application works as expected with this change. Consider that this represents a Major Breaking and non-backwards compatible change. It may break some thing we are working with and needs fully vetted.